packages/eas-cli/src/easMultiselect.ts

```diff
@@ -0,0 +1,55 @@
+import { Choice, PromptObject } from 'prompts';
+import { MultiselectPrompt } from 'prompts/lib/elements';
+
+const noop = (): void => {};
+
+export type Question<T extends string = string> = PromptObject<T> & {
+  selectionFormat?: string;
+};
+
+/**
+ * Customized multiselect prompt.
+ *
+ * Additional parameters:
+ *
+ * @param selectionFormat
+ *   String indicating number of selected options. Should contain `<num>` substring.
+ *
+ *   Example:
+ *     'Selected <num> devices'
+ *
+ *   Short format is used when more than one option is selected.
+ *
+ **/
+
+export default class EasMultiselect extends MultiselectPrompt {
+  constructor(opts: Question) {
+    super(opts);
+    this.selectionFormat = opts.selectionFormat;
+  }
+  override renderDoneOrInstructions(): string {
+    if (this.done && this.selectionFormat) {
+      const selectedOptionsCount = this.value!.filter(e => e.selected).length;
```

Comment:
Don't use `!`, prefer `nullthrows` or `assert`:
```suggestion
      assert(this.value);
      const selectedOptionsCount = this.value.filter(e => e.selected).length;
```